priv/static/Linux.sh

```diff
@@ -0,0 +1,368 @@
+
+        # recommendation after install homebrew
+        brew install gcc
+        ;;
+    "Asdf")
+        brew install asdf && echo -e "\n. $(brew --prefix asdf)/libexec/asdf.sh" >>${ZDOTDIR:-~}/.zshrc
+        ;;
+    "Erlang")
+        sudo apt-get update
+        sudo apt-get -y install build-essential autoconf m4 libncurses5-dev libwxgtk3.0-gtk3-dev libwxgtk-webview3.0-gtk3-dev libgl1-mesa-dev libglu1-mesa-dev libpng-dev libssh-dev unixodbc-dev xsltproc fop libxml2-utils libncurses-dev openjdk-11-jdk
+        asdf plugin add erlang https://github.com/asdf-vm/asdf-erlang.git
+        asdf install erlang 25.1.2 && asdf global erlang 25.1.2
+        ;;
+    "Elixir")
+        asdf plugin add elixir https://github.com/asdf-vm/asdf-elixir.git
+        asdf install elixir 1.14.2-otp-25
+        asdf global elixir 1.14.2-otp-25
+        asdf reshim elixir 1.14.2-otp-25
+        ;;
+    "Phoenix")
+        source ~/.bashrc >/dev/null 2>&1
+        source ~/.zshrc >/dev/null 2>&1
+        mix local.hex --force
+        echo "y" | mix archive.install hex phx_new 1.7.0-rc.2
+        ;;
+    "Node.js")
+        asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
+        asdf install nodejs 16.17.0
+        asdf global nodejs 16.17.0
+        asdf reshim nodejs 16.17.0
+        ;;
+    "PostgreSQL")
+        sudo apt-get update
+        sudo apt-get -y install linux-headers-generic build-essential libssl-dev libreadline-dev zlib1g-dev libcurl4-openssl-dev uuid-dev icu-devtools
+        asdf plugin add postgres https://github.com/smashedtoatoms/asdf-postgres.git
+        asdf install postgres 15.1
+        asdf global postgres 15.1
+        asdf reshim postgres 15.1
+        ;;
+    "Vim")
+        sudo apt-get install -y vim
+        ;;
+    "Chrome")
+        sudo wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+        sudo apt install -y ./google-chrome-stable_current_amd64.deb
+        ;;
+    "Chromedriver")
+        source ~/.bashrc >/dev/null 2>&1
+        source ~/.zshrc >/dev/null 2>&1
+        npm install -g chromedriver
+        ;;
+    "Docker")
+        sudo apt-get update
+        sudo apt-get install -y \
+            ca-certificates \
+            curl \
+            gnupg \
+            lsb-release
+        sudo mkdir -p /etc/apt/keyrings
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+        echo \
+            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+            $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list >/dev/null
+        sudo apt-get update
+        sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+        ;;
+    *)
+        echo "Invalid name argument on install"
+        ;;
+    esac
+}
+
+function get() {
```

Comment:
maybe_install()

Reply:
actually install() already defined as main function that do tools installation process, that's why I use get, maybe we need better word for it, or we could prefix it with underscore like `_install` maybe?